tests/operators/test_operators.py

```diff
@@ -0,0 +1,61 @@
+"""
+Unittest module to test Great Expectations Operators.
+
+Requires the unittest, pytest, and requests-mock Python libraries.
+pip install google-cloud-storage
+
+Run test:
+
+    python3 -m unittest test.operators.test_operators.TestGreatExpectationsOperator
+
+"""
+
+from pathlib import Path
+import logging
+import os
+import pytest
+import requests_mock
+import unittest
+from unittest import mock
+
+# Import Operator as module
+from great_expectations_provider.operators.great_expectations import GreatExpectationsOperator
+from great_expectations_provider.operators.great_expectations_bigquery import GreatExpectationsBigQueryOperator
+
+
+log = logging.getLogger(__name__)
+
+# Set relative paths for Great Expectations directory and sample data
+base_path = Path(__file__).parents[2]
+data_file = os.path.join(base_path,
+                         'include',
+                         'data/yellow_tripdata_sample_2019-01.csv')
+ge_root_dir = os.path.join(base_path, 'include', 'great_expectations')
+
+
+@mock.patch.dict('os.environ', AIRFLOW_CONN_MY_BIGQUERY_CONN_ID='http://API_KEY:API_SECRET@?extra__google_cloud_platform__key_path=extra__google_cloud_platform__key_path')
```

Comment:
Do we need this is we're only testing the base operator?

Reply:
You're right. Let's leave it out.